buildtool/Dockerfile

```diff
@@ -1,24 +1,15 @@
 # The docker image to build the binaries.
-FROM golang:1.5.3
+FROM golang:1.7
 MAINTAINER Yakau Bubnou <ybubnou@infoblox.com>
 WORKDIR /tmp
 
 # Set up mandatory Go environmental variables.
-ENV GO15VENDOREXPERIMENT=1
```

Comment:
Removed a ```GO15VENDOREXPERIMENT``` environment variable as in 1.7 version, "vendor" behavior is always enabled. Release notes: https://blog.golang.org/go1.7